trie/verkle.go

```diff
@@ -163,15 +162,19 @@ func (t *VerkleTrie) UpdateAccount(addr common.Address, acc *types.StateAccount,
 		stem      = t.pointCache.GetTreeKeyBasicDataCached(addr[:])
 	)
 
+	// XXX this is a workaround until we manage to "update subfields"
+	bd, err := t.root.Get(stem, t.FlatdbNodeResolver)
+	if err == nil && len(bd) > 0 {
+		basicData[utils.BasicDataVersionOffset] = bd[utils.BasicDataVersionOffset]
+		copy(basicData[utils.BasicDataCodeSizeOffset:utils.BasicDataNonceOffset], bd[utils.BasicDataCodeSizeOffset:utils.BasicDataNonceOffset])
```

Comment:
Couldn't we avoid this `r.root.Get` and fill these `basicData` with the `codeLen` parameter? Looks like we only use `bd` for the code-size.

I mean, in theory `codeLen` and `bd[utils.BasicDataCodeSizeOffset:utils.BasicDataNonceOffset]` should always be the same, if isn't we have a big problem?

Reply:
I had to check, and it seems indeed that the code will be loaded in any case, and therefore we can use `len(code)`. I'll change that.